mabar-frontend/src/services/aiConversationManager.ts

```
@@ -0,0 +1,406 @@
+    try {
+      // Import services dynamically to avoid circular dependencies
+      const { VenueService } = await import('./venueService')
+      const { PlayerService } = await import('./playerService')
+      const { SessionService } = await import('./sessionService')
+      
+      let response: any
+      
+      switch (action) {
+        case 'findVenues':
+          const venues = await VenueService.searchVenues(parameters)
+          response = { venues, totalResults: venues?.length || 0 }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)